configs/ballerina.json

```
@@ -2,18 +2,18 @@
   "index_name": "ballerina",
   "start_urls": [
     {
-      "url": "https://ballerina.io/learn",
+      "url": "https://ballerina.io",
       "tags": ["ballerina"],
       "extra_attributes": {
         "site": ["ballerina"]
       },
       "page_rank": 1
     },
     {
-      "url": "https://lib.ballerina.io/",
+      "url": "https://lib.ballerina.io",
       "page_rank": 2,
       "selectors_key": "swanlake_api_docs",
-      "tags": ["swanlake_api_docs","api","swanlake"],
+      "tags": ["swanlake_api_docs","api","swanlake","swan","lake"],
```

```suggestion
      "tags": ["swanlake_api_docs"],
```

Tags are added for filtering purposed, unless you have a use for all of these, you can restrict to `swanlake_api_docs` for example.

See https://docsearch.algolia.com/docs/config-file/#using-custom-tags